wallet/createtx.go

```diff
@@ -943,22 +935,67 @@ func (w *Wallet) purchaseTickets(op errors.Op, req purchaseTicketRequest) ([]*ch
 		return nil, errors.E(op, errors.Invalid, "stakepool fee percent unset")
 	}
 
+	inSizes := make([]int, 0)
+	outSizes := make([]int, 0)
+	// TODO: use stake.SStxPKHMinOutSize after it has been updated.
+	commitmentScriptSize := 34
```

Comment:
This can be a constant instead of a var.  There should also be a comment as to how it was calculated.  In fact, it could even go into txsizes with the other script size definitions